docker-compose.yml

```diff
@@ -8,8 +8,6 @@ services:
     ports:
       - "${HOST_MACHINE_UNSECURE_HOST_PORT}:80"
       - "${HOST_MACHINE_SECURE_HOST_PORT}:443"
-    links: 
-      - api-db
```

Comment:
Where is this linked? I couldn't find a reference?

Reply:
This would be the service name for the db. Can remove since we're going with SQLite :)